CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+Any contribution that you make to this repository will
+be under the Apache 2 License, as dictated by that
+[license](http://www.apache.org/licenses/LICENSE-2.0.html):
+
+~~~
```

Comment:
Maybe need delete this section?

Reply:
`CONTRIBUTING.md` is needed in ROS `ament_copyright`